app.R

```diff
@@ -517,13 +527,14 @@ schema_to_display_lookup <- data.frame(schema_name, display_name)
       })
       
       ### update DT view with incorrect values
-      ### currently only one column, requires backend support of multiple
-      output$tbl <- DT::renderDT({
-        datatable(rawData(),
-                    options = list(lengthChange = FALSE, scrollX = TRUE)
-          ) %>% formatStyle(errorDT$Column,
-                            backgroundColor = styleEqual(errorDT$Value, rep("yellow", length(errorDT$Value) ) )) ## how to have multiple errors
-      })
+      if (update_preview) {
+        output$tbl <- DT::renderDT({
+          datatable(rawData(),
+                      options = list(lengthChange = FALSE, scrollX = TRUE)
+            ) %>% formatStyle(errorDT$Column,
+                              backgroundColor = styleEqual(errorDT$Value, rep("yellow", length(errorDT$Value) ) ))
+        })
+      }
```

Comment:
Comment from @milen-sage: Highlight all columns in the preview table to "yellow". If it's visually appealing then retain it, else drop it. Also change the error message font color to "red".

Reply:
update in e845dde: highlight all cells to "yellow" if there is a 'wrong schema' error

Reply:
I will make a seperate pr to change error message to red in